google/cloud/spanner/samples/samples.cc

```diff
@@ -3142,6 +3147,92 @@ void RunAll(bool emulator) {
       run_slow_integration_tests.find("backup") != std::string::npos;
   auto const run_slow_instance_tests =
       run_slow_integration_tests.find("instance") != std::string::npos;
+
+  if (!run_slow_instance_tests) return;
```

Comment:
It is a shame that `GOOGLE_CLOUD_CPP_SPANNER_SLOW_INTEGRATION_TESTS=backup` is meaningless (you need to say `instance,backup`).  But, that's just an observation about the status quo ... nothing to do here.

Reply:
Ack.